docs/howto/operate/grafana.md

```diff
@@ -3,27 +3,76 @@
 
 Each 2i2c Hub is set up with [a Prometheus server](https://prometheus.io/) to generate metrics and information about activity on the hub, and each cluster of hubs has a [Grafana deployment](https://grafana.com/) to ingest and visualize this data.
 
-This section describes how to use these dashboards for a cluster.
+The following sections describe how access, use and deploy these dashboards for a cluster.
 
-## Access Hub Grafana Dashboards
+## 1. How to access and use the Grafana Dashboards
 
-The Grafana for each cluster can be accessed at `grafana.<cluster-name>.2i2c.cloud`.
-For example, the Grafana for community hubs running on our GCP project is accessible at `grafana.pilot.2i2c.cloud`.
+This section provides information for both engineers and non-engineers about where to find each of 2i2c Grafana deployments, how to get access and what to expect.
 
-To access the Grafana dashboards you'll need a **username** and **password**.
-These can be accessed using `sops` (see {ref}`tc:secrets:sops` for how to set up `sops` on your machine).
-See [](grafana:log-in) for how to find the credentials information.
+(grafana:access-grafana)=
+### Logging in
 
-## The Central Grafana
+Each cluster's Grafana deployment can be accessed at `grafana.<cluster-name>.2i2c.cloud`.
```

Comment:
You could probably add a link to [this page](https://infrastructure.2i2c.org/en/latest/reference/hubs.html) here since that provides all the grafana links

Reply:
Great idea! ty!